### PR TITLE
docs: correct link of github

### LINKED ISSUE #115: [DOCS] Sidebar GH link - change the URL
- [ ] The `Github` link in the sidebar, should point to [rn-notificated](https://github.com/TheWidlarzGroup/react-native-notificated)

### PR DESCRIPTION
**Pull request by ssavi-ict**
### Why
Closes #115 

### What Changed
- GitHub URL on the right sidebar is fixed with [rn-notificated](https://github.com/TheWidlarzGroup/react-native-notificated/blob/master/docs/docusaurus.config.js)

Please check my PR and validate if OK. 
I am new to OS contribution. I will be grateful for any feedback.
Thanks.

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -59,7 +59,8 @@ const config = {
           },
           // {to: '/blog', label: 'Blog', position: 'left'},
           {
-            href: 'https://github.com/facebook/docusaurus',
+            // href: 'https://github.com/facebook/docusaurus',
```
Comment (danielmark0116) [inline docs/docusaurus.config.js:62]:
```suggestion
```

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -59,7 +59,8 @@ const config = {
...
+			href: 'https://github.com/TheWidlarzGroup/react-native-notificated',
```
Comment (danielmark0116) [inline docs/docusaurus.config.js:63]: Please, format the code with prettier :)

**Review by danielmark0116 — CHANGES_REQUESTED** [review]
Thanks for your contribution and effort! Left 2 comments :)

File: docs/docusaurus.config.js (hunk 2 above)
Comment (ssavi-ict) [reply docs/docusaurus.config.js]: I have removed the facebook_docusaurus link and inserted spaces before rn_notificated to prettify.

Comment (danielmark0116) [reply docs/docusaurus.config.js]: 👍🏽

**Comment by ssavi-ict** [comment]
Hello @danielmark0116 , Can you please check the current title?
According to the mentioned link, the title should be consistent. I changed all the letters to lowercase.

Apologize for mistakes.

**Comment by danielmark0116** [comment]
Merging :) Thanks 👍🏽

**Pull request merged**